src/v/redpanda/admin_server.cc

```diff
@@ -560,6 +560,19 @@ void admin_server::register_status_routes() {
             {"status", _ready ? "ready" : "booting"}};
           return ss::make_ready_future<ss::json::json_return_type>(status_map);
       });
+
+    ss::httpd::status_json::started.set(
+      _server._routes, [this](std::unique_ptr<ss::httpd::request>) {
+          using futurator = ss::futurize<ss::json::json_return_type>;
+          // TODO(Ben): Determine a better proxy for started See #846
+          if (_ready) {
```

Comment:
shouldn't this be a controller function dispatch that asks controller if caught up with raft groups - for all raft groups?

Reply:
Thanks for the input. I'm actively seeking feedback on this, as I don't know the best way to do it, my high level thoughts are here: https://github.com/vectorizedio/redpanda/pull/1901#issuecomment-889320548